joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/utils/Scope.scala

```diff
@@ -18,6 +20,7 @@ class Scope(implicit nextClosureName: () => String) extends X2CpgScope[String, N
   private val anonymousMethods                                    = mutable.ArrayBuffer[Ast]()
   private var tmpVarCounter                                       = 0
   private var tmpClassCounter                                     = 0
+  private var importedSymbols                                     = Map.empty[String, Seq[SymbolSummary]]
```

Comment:
If the imported symbols are the same in a hole file and e.g. are not different from one method body to another, why does this need to live in the Scope object then?

Reply:
My plan for this is that the scope object can hide the logic that decides whether an identifier is a local variable or an import. In the former case, the scope object will be consulted so I figured I'd plan to combine the logic here instead.